sources/operator.1.4.0.pxc-operator.json

```diff
@@ -54,6 +54,14 @@
             "critical": false
           }
         },
+        "log_collector": {
```

Comment:
just Q: is it mandatory to add a new image to non-relevant versions?

Reply:
I though log collector can be used with any version of PXC.No?

Reply:
with any version of PXC - yes.
but only with >= 1.7.0 version of Operator

Reply:
https://github.com/Percona-Lab/percona-version-service/pull/26/commits/2ce22432b1ed32810692dd51675efb571f088b98